core/scripts/vrfv1/main.go

```diff
@@ -67,43 +62,43 @@ func main() {
 		D: d,
 	}
 	account, err := bind.NewKeyedTransactorWithChainID(&privateKey, big.NewInt(chainID))
-	panicErr(err)
+	helpers.PanicErr(err)
 
 	// Explicitly set gas price to ensure non-eip 1559
 	gp, err := ec.SuggestGasPrice(context.Background())
-	panicErr(err)
+	helpers.PanicErr(err)
 	account.GasPrice = gp
 
 	switch os.Args[1] {
 	case "ownerless-consumer-deploy":
 		cmd := flag.NewFlagSet("ownerless-consumer-deploy", flag.ExitOnError)
 		coordAddr := cmd.String("coordinator-address", "", "address of VRF coordinator")
 		linkAddr := cmd.String("link-address", "", "address of link token")
-		panicErr(cmd.Parse(os.Args[2:]))
+		helpers.ParseArgs(cmd, os.Args[2:], "coordinator-address", "link-address")
 		consumerAddr, tx, _, err := vrfoc.DeployVRFOwnerlessConsumerExample(
 			account,
 			ec,
 			common.HexToAddress(*coordAddr),
 			common.HexToAddress(*linkAddr))
-		panicErr(err)
+		helpers.PanicErr(err)
 		fmt.Printf("Ownerless Consumer: %s TX Hash: %s\n", consumerAddr, tx.Hash())
 	case "ownerless-consumer-request":
 		cmd := flag.NewFlagSet("ownerless-consumer-deploy", flag.ExitOnError)
```

Comment:
This flagset name is incorrect, it should be "ownerless-consumer-request" and not "ownerless-consumer-deploy". Seems like a copypasta error from before.